### PR TITLE
2017337 Update Tasks - Common - G4 - JavaToolInstallerV0

### PR DESCRIPTION
**Pull request by z00sts**
**Task name**: JavaToolInstallerV0 

**Description**: Upgrade from ADAL to MSAL

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** https://dev.azure.com/mseng/AzureDevOps/_workitems/edit/2017337

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/JavaToolInstallerV0/build_task.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+node make.js build --task JavaToolInstallerV0 && tfx build tasks delete --task-id C0E0B74F-0931-47C7-AC27-7C5A19456A36 && tfx build tasks upload --task-path .\_build\Tasks\JavaToolInstallerV0
```
Comment (max-zaytsev) [inline Tasks/JavaToolInstallerV0/build_task.sh:1]: Is this file added intentionally?

File: Tasks/JavaToolInstallerV0/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "java-tool-installer",
-    "version": "0.211.0",
+    "version": "0.215.0",
```
Comment (max-zaytsev) [inline Tasks/JavaToolInstallerV0/package.json:3]: We usually don't update the version here. You can either remove this or update the version in package-lock accordingly

File: Tasks/JavaToolInstallerV0/build_task.sh (hunk 1 above)
Comment (z00sts) [reply Tasks/JavaToolInstallerV0/build_task.sh:1]: I think it's better to remove it, thanks

File: Tasks/JavaToolInstallerV0/task.json [hunk 3]
```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 211,
-        "Patch": 0
+        "Minor": 216,
+        "Patch": 4
```
Comment (ozanovus) [inline Tasks/JavaToolInstallerV0/task.json:17]: Please don't forget to change Patch version to "0"

Comment (z00sts) [reply Tasks/JavaToolInstallerV0/task.json]: fixed

**Review by ozanovus — APPROVED** [review]

**Pull request merged**